lib/clients/AliClient.py

```diff
@@ -280,6 +280,9 @@ def _delete_snapshot(self, snapshot_id):
         except Exception as error:
             message = '{} ERROR: snapshot-id={}\n{}'.format(
                 log_prefix, snapshot_id, error)
+            if error.get_http_status() == 404:
+                self.logger.info(message)
```

Comment:
Do we need this logger with message "Not Found" ? Because message says `ERROR` which might be confusing.
Same applies to all other changes.

Reply:
As discussed, let's keep the message so that it's evident from the logs that this scenario occurred, but will modify the message to notify that this error is being ignored.